### ISSUE TITLE
Fix Inversify Error in Theia Diagram Extension

### ISSUE THREAD

**Issue by judgeNotFound** [schubmat/Custom-MADE#33]
An error occurs when trying to run an updated version of Theia. The error is already known and described in the Sprotty main repository [https://github.com/eclipse/sprotty/issues/201](https://github.com/eclipse/sprotty/issues/201). 

Full console log:
[https://pastebin.com/0KF4vjDm](https://pastebin.com/0KF4vjDm)

Possible Solution: Update to Sprotty 0.10.0

**Comment by schubmat** [comment]
Updated sprotty in b0e45484b254260fbf7315fede11aa4baa124905

see `/schubmat/Custom-MADE/tree/feature-integrate-visual-editing`

However, some compile errors remain. As soon as I have more time, I could have a loot at it. May be @rrcomtech or @pcprinz could have a look at it before that?

**Comment by judgeNotFound** [comment]
First error messages showing the error 
```
error TS2322: Type 'Element' is not assignable to type 'VNode'.
```
eliminated by f2dd1036cb4fe8875e28991aa2d5c039a6d833e4 (see [here](https://github.com/snabbdom/snabbdom/issues/985)). New errors remaining.

**Comment by judgeNotFound** [comment]
The error stated that modules are not present anymore in Sprotty-Theia `0.10.0`. It looked like this (for multiple modules):
```
src/browser/diagram/diagram-configuration.ts(3,10): error TS2305: Module '"sprotty-theia"' has no exported member 'CodeActionPalettePopupProvider'.
```
It was done, because many LSP-related modules were moved to [eclipse/sprotty-vscode](https://github.com/eclipse/sprotty-vscode). This topic is discussed in these issues:

- eclipse/sprotty-theia/issues/82
- eclipse/sprotty-theia/issues/67

To put it short: Theia does not support LSP-related modules anymore. The new approach is to implement a VSCode extension, which still does support LSP-related modules. 

An example for a reimplementation of a VSCode Sprotty extension is given in [eclipse/sprotty-vscode](https://github.com/eclipse/sprotty-vscode). The reimplementation should be the new main focus, which resolves this issue.

**Comment by judgeNotFound** [comment]
Fixed by 10d62d2af2a5e48b41574ca038dbeb05d9b726f1.
